lblod/app-digitaal-loket
# Form adjust: Afschrift Erkenningszoekende besturen

**[DL-6276]**

Form that needed adjustment: Afschrift Erkenningszoekende besturen

Some Afschrift Types needed to be added, and the alert banner in the form needed a rewording.

**Relies on:**

* https://github.com/lblod/manage-submission-form-tooling/pull/59
* https://github.com/lblod/ember-submission-form-fields/pull/196
* https://github.com/lblod/frontend-loket/pull/423

**How to test**

* Log in as an EB, for example "Kerkfabriek St.-Lambertus van Grobbendonk".
* Go to Toezicht and create a new submission.
* Select the type: "Afschrift erkenningszoekende besturen"
* Make sure that the banner is correct, and that the "Type afschrift" field shows the 4 new entries.
